02-oop-inheritance/javaScript/ver.0.2/main  v0.2.js

```diff
@@ -0,0 +1,58 @@
+var MovieObserver = function ( ) {};
+MovieObserver.prototype.playing = function (title) {
+    console.log('A movie with title ' + title + ' has been played');
+};
+MovieObserver.prototype.stopped = function(title) {
+    console.log('A movie with title ' + title + ' stopped');
+};
+
+
+var Movie = function (optParams) {
+  this.title = optParams.title;
```

Comment:
It's OK, leave it like this but anyway I prefer to provide multiple parameters to a constructor.